game-core/src/main/java/games/strategy/engine/chat/ChatTransmitter.java

```diff
@@ -1,18 +1,17 @@
 package games.strategy.engine.chat;
 
 import games.strategy.engine.lobby.PlayerName;
-import java.util.Map;
+import java.util.Collection;
 
 /**
  * Interface to represent 'send' actions from a chat client to server. This interface can be used to
  * push messages to the server.
  */
 public interface ChatTransmitter {
-  /**
-   * Connects to chat, adds current player and returns the set of players that are in chat mapped to
-   * their status.
-   */
-  Map<ChatParticipant, String> connect();
+  void setChatClient(ChatClient chatClient);
+
+  /** Connects to chat, adds current player and returns the set of players that are in chat. */
+  Collection<ChatParticipant> connect();
```

Comment:
Hmm are you sure it's good to go with a such a broad collection type?
I feel like specifying it's a set, makes it clear that there are no duplicate values.